src/main/java/bio/terra/service/upgrade/Migrate.java

```diff
@@ -76,4 +76,16 @@ public void migrateDatabase() {
         }
     }
 
+    /**
+     * Helper function to block "drop all on start" from happening on undesired databases
+     */
+    public boolean allowDropAllOnStart() {
```

Comment:
Now that it's no longer called from this class, it seems odd to have it here. Maybe move it to `JobService`?

Reply:
Ah, yeah - that would make sense. I've moved it!